core/app/[locale]/(default)/account/[tab]/layout.tsx

```diff
@@ -32,7 +32,7 @@ export default async function AccountTabLayout({ children, params: { locale, tab
   return (
     <NextIntlClientProvider locale={locale} messages={{ Account: messages.Account ?? {} }}>
       <AccountStatusProvider>
-        <h1 className="my-6 my-8 text-4xl font-black lg:my-8 lg:text-5xl">{t('heading')}</h1>
+        <h1 className="my-8 text-4xl font-black lg:my-8 lg:text-5xl">{t('heading')}</h1>
```

Comment:
do you mind to remove heading for tab layout if we do refactoring? Turner's updated figma and we don't need it anymore on Tabs.

Reply:
We're merging to a branch, will rebase when we have all changes ready to be merged in a final PR to main.